src/borg/repository.py

```diff
@@ -724,9 +724,51 @@ def __contains__(self, id):
         return id in self.index
 
     def list(self, limit=None, marker=None):
+        """
```

Comment:
would work, but I guess I'ld rather add a rpc method param and do try: newcall except ...: oldcall as we did before instead of introducing yet another way to deal with our rpc problems.